src/main/java/core/basesyntax/model/zoo/Animal.java

```diff
@@ -1,6 +1,23 @@
 package core.basesyntax.model.zoo;
 
+import java.util.Objects;
+import javax.persistence.DiscriminatorColumn;
+import javax.persistence.DiscriminatorType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name = "animal_kind",
+        discriminatorType = DiscriminatorType.INTEGER)
```

Comment:
Let's use String. You can say from 1 or 2 which one is cat and which is dog

Reply:
Fair, I made the changes